### ISSUE TITLE
ERROR: Cannot extend non-existing rule: "oas3-server-trailing-slash" since 6.0

### ISSUE THREAD

**Issue by acabarbaye** [stoplightio/spectral#1905]
When using the vanilla docker container from 6.0 onwards and carrying out `spectral lint ....yaml`:
I am getting the following error consistently:
`Cannot extend non-existing rule: "oas3-server-trailing-slash"`

This does not happen with version 5.9 or prior versions

**Comment by P0lip** [comment]
Does the ruleset you extend actually have that rule?

**Comment by acabarbaye** [comment]
Sorry I should have added my .spectral.yaml:
```
rules:
  oas3-server-trailing-slash: off
```

**Comment by P0lip** [comment]
Is that the entire ruleset you have?
In that case, you have no `extends` meaning the rule you try to override does not exist, and hence the error.
Prior to 6.x we silently ignored it, but as of 6.0 this becomes an actual exception.

**Comment by acabarbaye** [comment]
this rule is part of the built-in ruleset https://github.com/stoplightio/spectral/blob/develop/docs/reference/openapi-rules.md#oas3-server-trailing-slash and my goal was not to extend it but disable it. This worked well until 6.0.0

**Comment by P0lip** [comment]
That's right, but you don't extend the ruleset.
To do so, one has to do the following
```yaml
extends: spectral:oas
rules:
  oas3-server-trailing-slash: off
```

**Comment by acabarbaye** [comment]
I see. It looks like I had not fully understood the documentation then.
That solved the problem. Thank you for your help.
